packages/excalidraw/frame.ts

```diff
@@ -175,6 +190,7 @@ export const groupsAreCompletelyOutOfFrame = (
   groupIds: readonly string[],
   frame: ExcalidrawFrameLikeElement,
 ) => {
+  const elementsMap = arrayToMap(elements);
```

Comment:
likely hot path

Reply:
is this function used anywhere in the codebase ?